### ISSUE TITLE
Add support for IronRuby download from CodePlex

### ISSUE THREAD

**Issue by charlesroper** [vertiginous/pik#52]
IronRuby RC1 and future releases, presumably, are hosted on CodePlex. Need to add support for download from this source. See this thread for discussion:

http://groups.google.com/group/discuss_pik/browse_thread/thread/6fba58ce469293c4?hl=en

**Comment by jredville** [comment]
These should be on both RF and CP.

**Comment by klauern** [comment]
Has there been any progress update on this?  I just want to make sure that my error installing ironruby is related:

```
C:\Ruby-1.8.6-mswin32\pik>pik install ironruby
** Downloading:  http://rubyforge.org/frs/download.php/66606/ironruby-0.9.2.zip
   to:  C:\Documents and Settings\A03182\.pik\downloads\ironruby-0.9.2.zip

ironruby-0.9.2.zip: 100% |ooooooooooooooooooooo|   4.5MB/  4.5MB Time: 00:00:36

** Extracting:  C:\Documents and Settings\A03182\.pik\downloads\ironruby-0.9.2.zip
   to:  C:\Ruby-1.8.6-mswin32\pik\pik\IronRuby-092
done


There was an error.
 Error: private method `gsub' called for nil:NilClass

  in: pik/commands/command.rb:131:in `get_version'
  in: pik/commands/add_command.rb:23:in `add'
  in: pik/commands/add_command.rb:13:in `execute'
  in: pik/commands/install_command.rb:69:in `add'
  in: pik/commands/install_command.rb:25:in `execute'
  in: pik_runner:35
```

Has there been any progress update on this?  I just want to make sure that my error installing ironruby is related:

```
C:\Ruby-1.8.6-mswin32\pik>pik install ironruby
** Downloading:  http://rubyforge.org/frs/download.php/66606/ironruby-0.9.2.zip
   to:  C:\Documents and Settings\A03182\.pik\downloads\ironruby-0.9.2.zip

ironruby-0.9.2.zip: 100% |ooooooooooooooooooooo|   4.5MB/  4.5MB Time: 00:00:36

** Extracting:  C:\Documents and Settings\A03182\.pik\downloads\ironruby-0.9.2.zip
   to:  C:\Ruby-1.8.6-mswin32\pik\pik\IronRuby-092
done


There was an error.
 Error: private method `gsub' called for nil:NilClass

  in: pik/commands/command.rb:131:in `get_version'
  in: pik/commands/add_command.rb:23:in `add'
  in: pik/commands/add_command.rb:13:in `execute'
  in: pik/commands/install_command.rb:69:in `add'
  in: pik/commands/install_command.rb:25:in `execute'
  in: pik_runner:35
```

Update:  I guess I wasn't running the proper version of .NET.   I had 2.0, but 2.0SP1 was needed.  Not entirely intuitive from the exception, but it works now.

**Comment by aslakhellesoy** [comment]
1.0 has been out for a while, but pik doesn't seem to know how to install it. Also see #44 - I can't pik add it either.

**Comment by vertiginous** [comment]
fix IronRuby implementation regex - closed by a65a5bc4a679b78eac9e0a36de7a14a29a16fa2a

**Comment by vertiginous** [comment]
@klauren:
I agree, the pik error message isn't intuitive, but at least on Windows 7, I get a popup  with this message:

```
---------------------------
ir.exe - .NET Framework Initialization Error
---------------------------
To run this application, you first must install one of the following versions of the .NET Framework:

  v4.0.30319

Contact your application publisher for instructions about obtaining the appropriate version of the .NET Framework.
---------------------------
OK   
---------------------------
```
